lnrpc/rpc.proto

```diff
@@ -1515,6 +1515,21 @@ message Peer {
 
     /// Features advertised by the remote peer in their init message.
     map<uint32, Feature> features = 11 [json_name = "features"];
+
+    /*
+    Latest error is the last error we received from the peer. This error is
+    tracked across peer connections, but is not currently persisted across lnd
+    restarts.
+    */
+    TimestampError latest_error = 12;
+}
+
+message TimestampError{
+    // The unix timestamp in seconds when the error occurred.
```

Comment:
nit: strictly speaking, a unix timestamp is always in seconds.

Reply:
We do sometimes serialize timestamps as `UnixNano()` in `lnd` (I think) that's why usually the comment mentions seconds explicitly.

Reply:
Got a comment on another PR to explicitly indicate that it's in seconds, so now I just add it by default :)